apps/web/src/constants/routes.ts

```diff
@@ -49,15 +49,20 @@ export enum ROUTES {
   ORGANIZATION = '/settings/organization',
   SECURITY = '/settings/security',
   BILLING = '/settings/billing',
-  /** Novu V2 routes */
+
+  /** Novu V2 routes, defined as nested routes */
   STUDIO = '/studio',
-  STUDIO_FLOWS = '/studio/flows',
-  STUDIO_FLOWS_VIEW = '/studio/flows/:templateId',
-  STUDIO_ONBOARDING = '/studio/onboarding',
-  STUDIO_ONBOARDING_PREVIEW = '/studio/onboarding/preview',
-  STUDIO_ONBOARDING_SUCCESS = '/studio/onboarding/success',
-  STUDIO_FLOWS_STEP_EDITOR = '/studio/flows/:templateId/step/:stepId',
-  STUDIO_FLOWS_TEST = '/studio/flows/:templateId/test',
+  STUDIO_FLOWS = 'flows',
+  STUDIO_FLOWS_VIEW = 'flows/:templateId',
+  STUDIO_FLOWS_STEP_EDITOR = 'flows/:templateId/step/:stepId',
+  STUDIO_FLOWS_TEST = 'flows/:templateId/test',
+
+  STUDIO_ONBOARDING = 'onboarding',
+  STUDIO_ONBOARDING_PREVIEW = 'onboarding/preview',
+  STUDIO_ONBOARDING_SUCCESS = 'onboarding/success',
+
+  LOCAL_STUDIO = '/local_studio',
+  LOCAL_STUDIO_AUTH = '/local_studio/auth',
 }
 
-export const PUBLIC_ROUTES_PREFIXES = new Set<string>(['/auth']);
+export const PUBLIC_ROUTES_PREFIXES = new Set<string>(['/auth', '/local_studio']);
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Do we want `/local_studio/auth` instead? Or as well?‏